CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to Eclipse Tractus-X
+
+Thanks for your interest in this project.
+
+## Project description
+
+The companies involved want to increase the automotive industry's
+competitiveness, improve efficiency through industry-specific cooperation and
+accelerate company processes through standardization and access to information
+and data. A special focus is also on SMEs, whose active participation is of
+central importance for the network's success. That is why Catena-X has been
+conceived from the outset as an open network with solutions ready for SMEs,
+where these companies will be able to participate quickly and with little IT
+infrastructure investment. Tractus-X is meant to be the PoC project of the
+Catena-X alliance focusing on parts traceability.
+
+* https://projects.eclipse.org/projects/automotive.tractusx
+
+## Developer resources
+
+Information regarding source code management, builds, coding standards, and
+more.
+
+* https://projects.eclipse.org/projects/automotive.tractusx/developer
+
+The project maintains the source code repositories in the following GitHub organization:
```

Comment:
@flozf, license file and contributing file need changes.
Could you please adapt the change mentioned here in issue #2